### PR TITLE
ARTEMIS-1540 Missing management annotations

### PR DESCRIPTION
**Pull request by gaohoward**
Some of the management operations are not properly annotated.
Also need to add an Operation to list divert names.

### CODE REVIEW

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java [hunk 1]
```diff
@@ -631,7 +631,9 @@ default Message setAnnotation(SimpleString key, Object value) {
    default Map<String, Object> toPropertyMap() {
       Map map = new HashMap<>();
       for (SimpleString name : getPropertyNames()) {
-         map.put(name.toString(), getObjectProperty(name.toString()));
+         //some property is SimpleString, which is not available for management console
+         Object value = getObjectProperty(name.toString());
+         map.put(name.toString(), value == null ? null : value.toString());
```
Comment (michaelandrepearce) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/Message.java]: nice catch.

**Comment by michaelandrepearce** [comment]
+1 LGTM

**Comment by clebertsuconic** [comment]
Looks good.  I can’t merge it now. But it will do as soon as I can. (Or anyone can do it)

**Comment by michaelandrepearce** [comment]
Thanks @gaohoward merged.

**Comment by gaohoward** [comment]
thanks guys

**Pull request closed without merging**